src/components/inputs/file/file.component.tsx

```diff
@@ -81,9 +81,9 @@ const File: React.FC<FileProps> = ({ question, onChange, handler }) => {
   };
 
   useEffect(() => {
-    getConceptNameAndUUID(question.questionOptions.concept).then((conceptTooltip) => {
-      setConceptName(conceptTooltip);
-    });
+    if (question.meta) {
+      setConceptName(question.meta?.display);
+   }
```

Comment:
This is starting to look repetitive. Is there an option to get rid of the useEffect all together and just infer the question.meta?.display or is it tied to the question being fully loaded within the useEffect?

Reply:
+100

Reply:
thanks @pirupius i have removed the useEffect